snippets/csharp/VS_Snippets_CLR/conceptual.clscompliant/cs/naming1.cs

```diff
@@ -48,7 +48,7 @@ public class Size
 //    Naming2a.cs(13,8): (Location of symbol related to previous warning)
 // </Snippet17>
 
-// Å   Å    ������ 
```

Comment:
❓ What is this supposed to be? I was not able to figure out what the original intent here was.

Reply:
I think that it's somehow just garbage that was introduced into the file during migration. There's no snippet, it's not used in code, and I included an entry point only so that I wouldn't have to specify that I was compiling a library from the command line.

Reply:
Should I leave it like this, remove the part after the second Å, or remove the whole line?

Reply:
Just remove the whole line.